.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Mocha Tests",
+            "type": "pwa-node",
+            "request": "attach",
+            "port": 9229,
+            "continueOnAttach": true,
+            "autoAttachChildProcesses": false,
+            "resolveSourceMapLocations": [
+              "!**/node_modules/**",
+              "!**/.vscode/extensions/hbenl.vscode-mocha-test-adapter-*/**"
+            ],
+            "skipFiles": [
+              "<node_internals>/**"
+            ]
+          }
+    ]
+}
```

Comment:
nit: nice that you added this file to git, to help someone else debug tests. would you mind adding a doc or a loom video in doc to show how one can get that working for them?